.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+name: flake8
```

Comment:
I asked you to create a CI test for `black` and `flake8`, the first one is not here.